pyproject.toml

```
@@ -18,22 +16,27 @@ classifiers = [
     "Programming Language :: Python :: 3.11",
     "License :: OSI Approved :: BSD License",
     "Topic :: Software Development :: Libraries :: Python Modules",
-    "Topic :: Database :: Database Engines/Servers"
-]
-dependencies = [
-    "pyoxigraph~=0.3.14",
-    "rdflib>=6.3,<8.0"
+    "Topic :: Database :: Database Engines/Servers",
 ]
+dependencies = ["pyoxigraph~=0.3.14", "rdflib>=6.3,<8.0"]
 dynamic = ["version"]
 license = { text = "BSD-3-Clause" }
 name = "oxrdflib"
 readme = "README.md"
 requires-python = ">=3.7"
 
+
 [project.entry-points."rdf.plugins.store"]
-Oxigraph = "oxrdflib:OxigraphStore"
-OxMemory = "oxrdflib:OxigraphStore"
-OxSled = "oxrdflib:OxigraphStore"
+Oxigraph = "oxrdflib.store:OxigraphStore"
+OxMemory = "oxrdflib.store:OxigraphStore"
+OxSled = "oxrdflib.store:OxigraphStore"
+
+
+[project.entry-points."rdf.plugins.parser"]
+ox-turtle = "oxrdflib.parser:OxigraphTurtleParser"
+ox-n3 = "oxrdflib.parser:OxigraphNTriplesParser"
```

```suggestion
ox-ntriples = "oxrdflib.parser:OxigraphNTriplesParser"
ox-nt = "oxrdflib.parser:OxigraphNTriplesParser"
```

N3 is an other language: https://w3c.github.io/N3/spec/